racingcar/src/main/java/View/OutputView.java

```diff
@@ -0,0 +1,38 @@
+package View;
+
+import racingcar.Car;
+import java.util.List;
+import racingcar.Winner;
+
+public class OutputView {
+    public static final String bar = "-";
```

Comment:
```suggestion
    public static final String BAR= "-";
```
하지만 `bar`보다 더 좋은 네이밍이 있지 않을까요?